localstack-core/localstack/services/events/utils.py

```diff
@@ -130,6 +130,11 @@ def get_event_time(event: PutEventsRequestEntry) -> EventTime:
     return event_time
 
 
+def format_event_time(event_time: datetime) -> str:
```

Comment:
why would we need the same function 2 times? just use `event_time_to_time_string` ?